tests/tests/control_wam

```diff
@@ -1,46 +1,111 @@
 ###############################################################################
 #
-#  Global control multigases test
+#  Global fv3 wam RT test
 #
 ###############################################################################
 
-export TEST_DESCR="Compare global control multigases results with previous trunk version"
+export TEST_DESCR="fv3 wam test"
 
 export CNTL_DIR=control_wam
 
 export LIST_FILES="sfcf024.nc \
                    atmf024.nc"
 
 export_fv3_v16
+export_tiled
```

Comment:
Why are the v16 defaults used (`export_fv3_v16`) if the wam is now running v17?

Reply:
Yes, this should be changed to _export_fv3_. Same with the control_wam_debug test file.

Reply:
Yes, this should be changed.

Reply:
@WChen-NOAA Can you update your branch to reflect the change to **_export_fv3_** and test to make sure that your version of control_wam and control_wam_debug still work with that call?

Reply:
@DeniseWorthen I don't know how to reflect it. but it is in the commit "56295a638". And I tested it yesterday. The wam RTs still work.